### DISCUSSION TITLE
Use export default instead sub export

### DISCUSSION THREAD

**Discussion by lamualfa** [tw-in-js/twind#25] [Q&A] [answered]
Wouldn't it be more concise if we did the import statement like this:

```js
import tw from 'twind'
```

Instead

```js
import { tw } from 'twind'
```

I honestly don't know the reason why you guys did that.

Thanks.

**Comment by sastan** [comment, marked as answer]
Thanks for the question. We are not using the default export for several reasons:

- we have several named exports and types 
- we are providing CJS and UMD bundles that are difficult to implement when having mixed exports
- if you start writing  `tw` in an editor most can auto-import it if it is a named export

From our view these outweigh the need to write two additional curly braces.

I hope this did answer your question - otherwise please follow up
